project.go

```diff
@@ -97,6 +97,72 @@ type (
 		ServiceName string `json:"service_name"`
 		Time        string `json:"time"`
 	}
+
+	// ProjectServiceTypesResponse is the response from Aiven for listing project service types
+	ProjectServiceTypesResponse struct {
+		APIResponse
+		ServiceTypes map[string]*ServiceType `json:"service_types"`
```

Comment:
Can we use here `map[string]ServiceType`? Since map or slice are all reference types in Golang, it is unlikely that we would have service type X have an empty set of user configs.

Reply:
and so this is 😅